src/guides/v2.3/config-guide/prod/config-reference-systemxml.md

```diff
@@ -0,0 +1,358 @@
+
+## Tabs
+
+A `<tab>`-Tag references to either an existing or a new tab in the system configuration.
+
+### Tab attribute reference
+
+A `<tab>`-Tag can have the following attributes:
+
+| Attribute   | Description                                                                                                                                         | Type     | Required |
+|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|----------|----------|
+| `id`        | Defines the identifier that is used referencing the section.                                                                                        | `typeId` | required |
+| `translate` | Defines the field that should be translatable. Provide `label` to make the label translatable.                                                      | `string` | optional |
+| `type`      | Defines the input type of the rendered HTML element—defaults to `text`.                                                                             | `string` | optional |
+| `sortOrder` | Defines the sort order of the section. High numbers push the section to the bottom of the page; low numbers drive the section to the top.           | `float`  | optional |
+| `class`     | Adds a defined CSS class to the rendered tab HTML element.                                                                                          | `string` | optional |
+
+### Tab node reference
+
+A `<tab>`-Tag can have the following child:
+
+| Node    | Description                                          | Type     |
+|---------|------------------------------------------------------|----------|
+| `label` | Defines the label that is displayed in the frontend. | `string` |
+
+### Example: Create a new tab
+
+The following code snippet demonstrates the creation of a new tab with example data.
+
+``` xml
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="A_UNIQUE_ID" translate="label" class="a-custom-css-class-to-style-this-tab" sortOrder="10">
+            <label>A meaningful label</label>
+        </tab>
+    </system>
+</config>
+```
+
+The snippet above creates a new tab with the identifier `A_UNIQUE_ID`. As the `translate`-attribute is defined and references the label, the `label`-node is translatable. During the rendering process, the css class `a-custom-css-class-to-style-this-tab` will be applied on the HTML element that was created for this tab.
+The `sortOrder`-attribute with the value of `10` defines the position of the tab in the list of all tabs when rendered.
+
+## Sections
+
+A `<section>`-Tag references to either an existing or a new section in the system configuration. The following screenshot shows a rendered section in the Magento Admin:
```

Comment:
There's no image reference here. Is that an oversight?

Reply:
Yes, right. Initially, I started to create multiple screenshots for each node but I ended up in the single screenshot showing all the different rendered nodes in one. Will remove this sentence.